meteor/server/api/rest/v1/index.ts

```diff
@@ -92,6 +93,19 @@ function extractErrorDetails(e: unknown): string[] | undefined {
 	}
 }
 
+export const checkValidation = (method: string, configValidationMsgs: IConfigMessage[]): void => {
+	const configValidationOK = configValidationMsgs.reduce((acc, msg) => acc && msg.level === NoteSeverity.INFO, true)
```

Comment:
What about `NoteSeverity.WARNING`? My initial assumption would be to still allow them, but log the warnings or return the warnings to the caller?

Reply:
Notes from a chat with Simon: Let's consider WARNINGs to be "okay" but log the warnings somewhere.